### PR TITLE
fix: proper OpenShift Proxy URL for preview environment

### PR DESCRIPTION
**Pull request by bartoszmajsak**
Based on https://github.com/openshiftio/openshift.io/issues/4125 Openshift Proxy has a new URL right now.

### CODE REVIEW

**Comment by gastaldi** [comment]
It seems that the tests are failing with this PR. Meanwhile, can you squash into a single commit?

**Comment by bartoszmajsak** [comment]
> It seems that the tests are failing with this PR

```
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 6.374 s <<< FAILURE! - in io.fabric8.launcher.osio.steps.OpenShiftStepsTest
[ERROR] shouldCreateBuildConfig(io.fabric8.launcher.osio.steps.OpenShiftStepsTest)  Time elapsed: 4.245 s  <<< ERROR!
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://osoproxy.prod-preview.openshift.io/.
```

Is that really what we want in the tests? To talk against real instance?

> Meanwhile, can you squash into a single commit?

How about using "squash and merge" from GH? It takes PR title as a commit msg. And keeps review history sane.

**Comment by bartoszmajsak** [comment]
Ok I found the problem. Hoverfly doesn't handle new URL.

**Comment by gastaldi** [comment]
OpenShiftStepsTest has references to the old host (in the Hoverfly JSON)

File: addons/osio-addon/src/test/java/io/fabric8/launcher/osio/steps/OpenShiftStepsTest.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
 public class OpenShiftStepsTest {
 
     private static final HoverflyRule HOVERFLY_RULE = createHoverflyProxy("openshiftsteps-simulation.json",
-                                                                          "github.com|githubusercontent.com|api.openshift.io|api.prod-preview.openshift.io|openshiftapps.com");
+                                                                          "github.com|githubusercontent.com|api.openshift.io|prod-preview.openshift.io|openshiftapps.com");
 
     @ClassRule
     public static RuleChain ruleChain = RuleChain
```
Comment (gastaldi) [inline addons/osio-addon/src/test/java/io/fabric8/launcher/osio/steps/OpenShiftStepsTest.java]: Can you change the `LAUNCHER_MISSIONCONTROL_OPENSHIFT_API_URL` default value here as well?

Comment (bartoszmajsak) [reply addons/osio-addon/src/test/java/io/fabric8/launcher/osio/steps/OpenShiftStepsTest.java]: Done in 0b2f8a5

**Comment by gastaldi** [comment]
Thanks!

**Pull request merged**